### DISCUSSION TITLE
Is there a way i can enable bottom navigation bar?

### DISCUSSION THREAD

**Discussion by vlkvch** [bromite/bromite#1169] [Q&A] [answered]
I've tried to search for "Chrome Home" and "Duet", but these options aren't available.

**Comment by ghost** [comment, marked as answer]
> I've tried to search for "Chrome Home" and "Duet", but these options aren't available.

I'm also looking for bottom navigation bar. It make the UX more accessible.

Unfortunately, the flag was removed in some version of Chrome that I don't remember which was.

**Reply by vlkvch** [reply]
Thank you.
